.storybook/preview-body.html

```diff
@@ -1,5 +1,6 @@
 <style>
     #root {
+        height: 100vh;
```

Comment:
May be safer to do

```
html,
body,
#root {
    height: 100%;
}

Reply:
Also, is this going to affect visual regression snapshots?

Reply:
It may, but I'm not sure yet. Will keep an eye on Travis